csrc/ops/arith.cpp

```diff
@@ -561,15 +571,14 @@ TensorView* eye(Val* size, DataType dtype) {
 
 // UNARY OPERATIONS
 
-#define NVFUSER_DEFINE_UNARY_OP(op_name, op_type) \
-  Val* op_name(Val* v) {                          \
-    return unaryOp(UnaryOpType::op_type, v);      \
-  }                                               \
-  TensorView* op_name(TensorView* tv) {           \
-    return unaryOp(UnaryOpType::op_type, tv);     \
+#define NVFUSER_DEFINE_UNARY_OP(operator_name, operator_type) \
```

Comment:
I am using clang-format 15, which format this differently from clang-format 12. This is super annoying for me, because I need to "unformat" this everytime. So I change this macro to use looooooong names, the names should be long enough so that clang-format 15 won't want to put the function definitions in one line.